### ISSUE TITLE
no interesting inputs were found

### ISSUE THREAD

**Issue by shouc** [ant4g0nist/ManuFuzzer#3]
Hello, thanks for the nice work. I am trying to run the example provided in the README. However, I got following error:

```
INFO: Running with entropic power schedule (0xFF, 100).
INFO: Seed: 3835002896
INFO: 65536 Extra Counters
INFO:        1 files found in seeds
INFO: -max_len is not provided; libFuzzer will not generate inputs larger than 4096 bytes
INFO: seed corpus: files: 1 min: 2b max: 2b total: 2b rss: 50Mb
#2      INITED exec/s: 0 rss: 50Mb
ERROR: no interesting inputs were found. Is the code instrumented for coverage? Exiting.
```

It appears that the instrumentMe functions have successfully instrumented the libraries based on the following log:

```
....
branch: true addr: 1baf5488c origByte: 9028d211 shadow: 3baf5488c
branch: true addr: 1baf5489c origByte: 9028d211 shadow: 3baf5489c
branch: true addr: 1baf548ac origByte: 9028d211 shadow: 3baf548ac
done
instrumentMe /System/Library/PrivateFrameworks/FontServices.framework/libFontParser.dylib done
```

I tried to use TinyInst to do the instrumentation for the example program and collect the BB coverage. It works fine for the compiled example binary. 

```
./TinyInst/litecov -instrument_module CoreText -target-env DYLD_INSERT_LIBRARIES=/usr/lib/libgmalloc.dylib -patch_return_addresses -coverage_file cov.txt -- ./bin/example  
```

I have tested it on M1 & Intel Macs running macOS 12.1 and 13.2, and none works. Can you please help?

**Comment by ant4g0nist** [comment]
Hey, thank you for raising the issue. As I understand, TinyInst instruments every library, while in Manufuzzer we are targeting to the libraries that we specifically instrumented using `instrumentMe`.

You are right that when we run the example without passing in any seeds, none of the instrumented libraries are being hit. I just passed a single ttf file as seed and it started finding paths. I have tested this on my M1.

<img width="1275" alt="Screenshot 2023-01-31 at 06 50 22" src="https://user-images.githubusercontent.com/3500559/215687327-adae7abb-8ab1-43fb-8f46-fea89f07e3b3.png">


Hope this answers your question.
